src/styles/index.ts

```diff
@@ -4045,6 +4045,20 @@ const styles = (theme: ThemeColors) =>
             width: 1,
         },
 
+        emptySageIntacctUserDimensionsSubtitle: {
+            fontSize: variables.fontSizeNormal,
+            lineHeight: variables.lineHeightXLarge,
+            color: theme.textSupporting,
+            textAlign: 'center',
+        },
+
+        sageIntacctUserDimensionsSubtitle: {
```

Comment:
Same comment as above - this all feels like super specific styles. Why do we need them? Can you show me a screenshot of what they do exactly?